tests/managers.py

```diff
@@ -4,4 +4,11 @@
 
 class SubjectQuerySet(models.QuerySet):
     def from_dicom_patient(self, patient: Patient) -> tuple:
-        return self.create(), True
+        data = {
+            "id_number": patient.uid,
+            "first_name": patient.given_name,
+            "last_name": patient.family_name,
+            "date_of_birth": patient.date_of_birth,
+            "sex": patient.sex,
+        }
+        return self.get_or_create(**data)
```

Comment:
General note: pay attention to docstrings. It's beneficial even to include a superficial docstring as a placeholder for future explanations or examples.